src/command_modules/azure-cli-role/azure/cli/command_modules/role/custom.py

```diff
@@ -807,6 +807,34 @@ def delete_permission(cmd, identifier, api):
     return graph_client.applications.patch(application.object_id, update_parameter)
 
 
+def admin_consent(cmd, identifier):
+    import requests
+    from azure.cli.core.cloud import AZURE_PUBLIC_CLOUD
+    from azure.cli.core._profile import Profile
+    from azure.cli.core.commands.client_factory import UA_AGENT
+    from azure.cli.core.util import should_disable_connection_verify
+    if cmd.cli_ctx.cloud.name != AZURE_PUBLIC_CLOUD.name:
+        raise CLIError('This command does not function yet in solverign clouds')
```

Comment:
typo! "solverign" should be "sovereign"

nit, I'd rephrase this to "This command is not yet supported on sovereign clouds"